### PR TITLE
Modifies icon feedback class to eliminate the need for window

### PR DESCRIPTION
**Pull request by jaysaleh**
Before, this class was using window to access variables, this eliminates that need.

### CODE REVIEW

File: src/main/webapp/feedback/post-icon-feedback.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+/** Stores icon feedback data in the database. */
+export default class PostIconFeedback {
+  static #ENDPOINT_FEEDBACK = '/icon-feedback';
+  static #PARAM_LECTURE_ID = 'lectureId';
+  static #PARAM_TIMESTAMP = 'timestampMs';
+  static #PARAM_ICON_TYPE = 'iconType';
+
+  #lecture;
+  #video
```
Comment (n-wach) [inline src/main/webapp/feedback/post-icon-feedback.js:23]: add `;` to be consistent

File: src/main/webapp/view/lecture-view.js [hunk 2]
```diff
@@ -13,6 +13,8 @@
 // limitations under the License.
 
 import EventController from '../event-controller.js';
+import PostIconFeedback from '../feedback/post-icon-feedback.js';
+
```
Comment (n-wach) [inline src/main/webapp/view/lecture-view.js:17]: try removing this line?

File: src/main/webapp/feedback/post-icon-feedback.js (hunk 1 above)
Comment (jaysaleh) [reply src/main/webapp/feedback/post-icon-feedback.js:23]: thanks for catching that

File: src/main/webapp/feedback/post-icon-feedback.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Stores icon feedback data in the database. */
+export default class PostIconFeedback {
+  static #ENDPOINT_FEEDBACK = '/icon-feedback';
+  static #PARAM_LECTURE_ID = 'lectureId';
+  static #PARAM_TIMESTAMP = 'timestampMs';
+  static #PARAM_ICON_TYPE = 'iconType';
```
Comment (f-camp) [inline src/main/webapp/feedback/post-icon-feedback.js:20]: nit: is it possible to make these static and const?

File: src/main/webapp/feedback/post-icon-feedback.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Stores icon feedback data in the database. */
+export default class PostIconFeedback {
+  static #ENDPOINT_FEEDBACK = '/icon-feedback';
+  static #PARAM_LECTURE_ID = 'lectureId';
```
Comment (f-camp) [inline src/main/webapp/feedback/post-icon-feedback.js:18]: nit: is it possible to make these static and const?

Comment (jaysaleh) [reply src/main/webapp/feedback/post-icon-feedback.js:18]: No in javascript classes

**Pull request merged**